i18n/english.yml

```diff
@@ -450,6 +450,23 @@ components:
     placeholder: Select language...
   Login:
     title: Log in
+  MergeFeedsResult:
+    body:
+      failure: Merge failed with $errorCount errors.
```

Comment:
This is inconsistent with the way template strings are done in `components.FeedTransformationDescriptions.DeleteRecordsTransformation`. Can we agree upon a consistent way of doing template strings?

Reply:
Addressed in https://github.com/ibi-group/datatools-ui/pull/686/commits/1c1239097f50b08c9c2172d9de671f91ae96cc77.